templates/cros-ec/generic-uboot-tftp-ramdisk-cros-ec-template.jinja2

```diff
@@ -0,0 +1,7 @@
+{% extends 'boot/generic-uboot-tftp-ramdisk-template.jinja2' %}
```

Comment:
Maybe we could run this one on `peach-pi` (U-Boot)?

Reply:
Done, test run at https://lava.collabora.co.uk/scheduler/job/1184534